components/molecules/FilterHeader/filter-header.tsx

```diff
@@ -40,15 +36,14 @@ const HeaderFilter = () => {
   return (
     <>
       <div className="header-image mr-2 p-2 min-w-[130px] ">
-        <ContextThumbnail size={120} ContextThumbnailURL={isHacktoberfest ? Thumbnail.src : ""}></ContextThumbnail>
+        <ContextThumbnail size={120} ContextThumbnailURL={getTopicThumbnail(filterName as string)}></ContextThumbnail>
       </div>
       <div className="header-info md:truncate flex flex-col grow justify-center p-2">
         <Title level={1} className="!text-3xl font-semibold tracking-tight text-slate-900">
-          {isHacktoberfest ? "Hacktoberfest 2022" : topicNameFormatting(filterName as string)}
+          {topicNameFormatting(filterName as string)}
         </Title>
         <Text className="mt-1 !text-base   text-slate-500">
-          Insights on GitHub repositories{" "}
-          {isHacktoberfest ? "opted into the largest open source hackathon." : `using the ${filterName} topic.`}
+          {`Insights on GitHub repositories using the ${filterName} topic.`}
```

Comment:
```suggestion
          {`Insights on GitHub repositories using the ${topicNameFormatting(filterName as string)} topic.`}
```

Reply:
If consistent images are needed, we can get @getaheaddev to do that in figma

Reply:
Since these are SVG, can we achieve it by providing width and height in the SVG file? Also, [ContextThumbnailProps](https://github.com/open-sauced/insights/blob/beta/components/atoms/ContextThumbnail/context-thumbnail.tsx) takes `size` as its optional props, so we can also set a boundary for the size.

@bdougie @brandonroberts